elixir/lib/app_identity/plug.ex

```diff
@@ -190,19 +190,29 @@ if Code.ensure_loaded?(Plug.Conn) do
         plug AppIdentity.Plug, on_failure: :continue
         ```
 
-      - A 1-arity anonymous function or a {module, function} tuple accepting
+      - A 1-arity anonymous function or a `{module, function}` tuple accepting
```

Comment:
(For later) The `option` type’s documentation is getting long and convoluted. This can be a good thing as it keeps stuff together, but it can also lead to misreading. I would like to do an experiment later to see what it looks like if you do something like

```
@type option() ::
  {:apps, apps_option_type}
  | {:disallowed, disallowed_option_type}
  | {:finder, finder_option_type}
  | {:headers, headers_option_type}
  | {:header_groups, header_groups_option_type}
  | {:name, atom()}
  | {:after, conn_fn} // grouped here, not alphabetically
  | {:on_failure, on_failure_option_type}
  | {:on_success, conn_fn}

At least one of `apps` or `finder` **must** be supplied. If both are present, apps are looked up in the `apps` list first.

Only one of `headers` or `header_groups` may be supplied. If both are present, an exception will be raised.

mermaid flow chart showing the relationship of on_x and after
```